jobs/build/send-umb-messages/Jenkinsfile

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env groovy
+
+node {
+    properties([
+        disableConcurrentBuilds(),
+    ])
+
+    // we only care to publish messages for the following releases
+    releases = ["4-stable", "4.2.0-0.nightly"]
+
+    stage("send UMB messages for new releases") {
+        dir ("~/.cache/releases") {
+            for (String release : releases) {
+                latestRelease = sh(
+                    returnStdout: true,
+                    script: "curl -s https://openshift-release.svc.ci.openshift.org/api/v1/releasestream/${release}/latest",
+                ).trim()
+                previousRelease = readFile("${release}.current")
+                if ( latestRelease != previousRelease ) {
+                    sendCIMessage(
+                        messageContent: "New release payload for OpenShift ${release}",
+                        messageProperties: "${latestRelease}",
+                        messageType: 'ProductAcceptedForReleaseTesting',
```

Comment:
@xltian @akostadinov do we know what message type to use for these?

Reply:
reference this: https://docs.google.com/document/d/1Vhc7SCw4bK2H_gUWAlDbQn-fl3iyq0JFM6zN8vgBKTw/edit?ts=5d3b4990#heading=h.otfn201g4q2j

Reply:
The example there is 

```json
{
  "contact": {
    "url": "https://mojo.redhat.com/docs/DOC-1201884",
    "team": "OpenShift Automatic Release Team (ART)",
    "email": "aos-team-art@redhat.com",
    "name": "ART Jobs",
    "slack": "#aos-art"
  },
  "run": {
    "url": "https://some-private-jenkins.com/job/ocp4-wait-for-release-controller/99",
    "log": "https://some-private-jenkins.com/job/ocp4-wait-for-release-controller/99/console",
    "additional_urls": {
        "prow": "https://openshift-release.svc.ci.openshift.org/releasestream/4.1.0-0.ci/release/4.1.0-0.ci-2019-07-25-212038",
        "aws": "https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/logs/release-openshift-origin-installer-e2e-aws-4.1/994",
        "aws-serial": "https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/logs/release-openshift-origin-installer-e2e-aws-serial-4.1/988",
        "upgrade": "https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/logs/release-openshift-origin-installer-e2e-aws-upgrade/4646"
    }
  },
  "artifact": {
    "type": "product-build",
    "name": "openshift",
    "version": "4.1.0",
    "release": "0.ci-2019-07-25-212038",
    "nvr": "openshift-4.1.0-0.ci-2019-07-25-212038",
    "id": "sha256:61ee5a498914a539873ea519794c309440fc34bd2c1e8693dcce35b7e86f4aa6",
    "artifacts": [
        {
            "type": "container-image",
            "id": "sha256:65010529d088f9f0bb9f3115f676367846653522547035e8100b06ece79ec22d",
            "digest": "sha256:65010529d088f9f0bb9f3115f676367846653522547035e8100b06ece79ec22d",
            "pull_ref": "docker://registry.svc.ci.openshift.org/ocp/release@sha256:65010529d088f9f0bb9f3115f676367846653522547035e8100b06ece79ec22d",
            "repository": "ocp/release",
            "issuer": "tbielawa"
        },
        {
            "type": "uncontrolled-build",
            "id": "sha256:9259666990bb8e67402de7b76806f5bb214200aae14892ecc5011ef8c9edcb40",
            "url": "https://openshift-release-artifacts.svc.ci.openshift.org/4.1.0-0.ci-2019-07-25-212038/sha256sum.txt",
            "mime_type": "application/text",
            "issuer": "tbielawa"
        }
    ]
  },
  "test": {
    "type": "acceptance",
    "category": "validation",
    "namespace": "art"
  },
  "generated_at": "2019-07-26 00:01:00.000000",
  "version": "0.2.1"
}
```

That requires quite a lot more work including but not limited to changes to the release controller to add new JSON APIs for the data they want in that message. I don't think I can commit to that.

Reply:
As I proposed in a comment, we can also generate a `container-image` artifact type only. Not a `product-build` artifact with image details inside. Perhaps it would have less fields to figure out.

Reply:
@akostadinov @xltian what are you doing with the derivative events? Are you taking the output artifacts and testing them? Or are you actually looking back to check the output of the jobs that promoted them in the first place? You *could* go to the human-facing HTML pages to look at the old jobs, but I would like to understand what the use-case is here a little better first.

Reply:
@jianlinliu Please review and chime in about what information is needed from UMB to setup the cluster .

Reply:
I do not sure about UMB stuff, maybe @akostadinov should know it.

Reply:
I think I derailed this when I linked that document. My mistake. 

The original idea for the card that steve is working on was that this would be a temporary hack-job to get us by until we can work towards the ideas as they're described in the drive document. I wouldn't expect this to comply with those schemas right now. I referenced the document because I thought it had the name of the message type.

The question we need to answer is, is QE ok with this setting the `messageType` to `product-accepted-for-release-testing` if we don't comply with any possible schema. **OR** should we just use the `custom` type right now?

As @jianlinliu points out, we just need to send the payload url presently, which is pretty much what steve has written up in this PR.

Reply:
Yes, it seems like we need _some_ `messageType` to send the data we have with. The data we have is _not_ conformant to the spec in the doc, so whatever we choose will not follow it today.

Reply:
@stevekuznetsov  let's go with `custom` for now. ART can change it if that doesn't work out well.